src/com/facebook/buck/cxx/LinkableCxxConstructorArg.java

```diff
@@ -17,13 +17,17 @@
 package com.facebook.buck.cxx;
 
 import com.facebook.buck.cxx.toolchain.linker.Linker;
-import java.util.Optional;
+import com.facebook.buck.rules.SourcePath;
 import org.immutables.value.Value;
+import java.util.Optional;
 
 public interface LinkableCxxConstructorArg extends CxxConstructorArg {
   Optional<Linker.LinkableDepType> getLinkStyle();
 
   @Value.Default
+  Optional<SourcePath> getStaticPicLib();
```

Comment:
Sorry, not sure I follow this.  Why are we adding a `static_pic_lib` parameter to *all* C/C++ rules (via `LinkableCxxConstructorArg`)?  Shouldn't this change only affect `prebuilt_cxx_library`?